Loop/Extensions/NSUserDefaults+StatusExtension.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  NSUserDefaults+StatusExtension.swift
+//  Loop
+//
+//  Created by Bharat Mediratta on 11/27/16.
+//  Copyright © 2016 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+extension UserDefaults {
+    
+    private enum Key: String {
+        case StatusExtensionContext = "com.loudnate.Loop.StatusExtensionContext"
+    }
+
+    static func shared() -> UserDefaults? {
```

Comment:
This should be a class var like `standard` is rather than a static function. Also, `shared` isn't descriptive enough IMO. `group` or `appGroup` would make more sense.